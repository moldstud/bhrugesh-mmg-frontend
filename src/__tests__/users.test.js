import React from 'react';
import { shallow } from 'enzyme';
import UserList from '../pages/userList';

describe('User List', () => {
	it('should render correctly in "debug" mode', () => {
		const component = shallow(<UserList debug />);

		expect(component).toMatchSnapshot();
	});
});
