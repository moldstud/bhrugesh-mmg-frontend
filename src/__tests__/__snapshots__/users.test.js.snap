// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User List should render correctly in "debug" mode 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
